main.html

```diff
@@ -95,7 +95,9 @@
       <button id="clearstorage">Clear storage</button>
       <button id="exportstorage">Export storage</button>
       <div>Import storage:</div>
-      <input type="file" id="importstorage">
+      <input type="file" id="importstoragefile">
+      <input type="text" id="importstorageurl">
```

Comment:
Nit: the field could use a placeholder attribute to suggest that it takes a URL.